src/Laravel/TermwindServiceProvider.php

```diff
@@ -15,7 +16,9 @@ final class TermwindServiceProvider extends ServiceProvider
     public function register(): void
     {
         if ($this->app->runningUnitTests()) {
-            Termwind::renderUsing(new TestingBufferedOutput($this->app));
+            $this->app->resolving(OutputStyle::class, function ($style): void {
+                Termwind::renderUsing($style->getOutput());
+            });
         }
```

Comment:
Figure that we can take advantage of the resolving event.

Reply:
That way we get the mocked version from the `PendingCommand`

Reply:
Perfect, last question: why only do this in tests?

Reply:
Thats right, It works well for all, on Testing it gets the mocked version from `PendingCommand` and from the `php artisan command` it gets the `ConsoleOutput`.